spec/support/request_specs.rb

```diff
@@ -1,11 +1,26 @@
+require 'rack/test'
+require 'json_spec'
```

Comment:
You are right. It works without the `require` statements. I've just reverted my commit. Good to merge?